lib/solr.ts

```diff
@@ -880,6 +880,40 @@ export class Client {
   ping(callback: CallbackFn) {
     return this.get(this.ADMIN_PING_HANDLER, callback);
   }
+
+  createSchemaField(fieldName, fieldType, cb) {
+    const payload = {
+      'add-field': {
+        name: fieldName,
+        type: fieldType,
+        multiValued: false,
+        stored: true,
+      },
+    };
+
+    const params = {
```

Comment:
can we type this? is this Request options or something from http module?

Reply:
I'm not exactly sure. It's was only used in tests. I'll have a look.

Reply:
This is the argument expected by postJSON, which I will remove / replace / properly type in a next PR.

Remember, I'm only **moving** a method in this PR, not rewriting or even refactoring it.

Reply:
It's the same as here https://github.com/lbdremy/solr-node-client/blob/master/lib/solr.ts#L594